data/fields/highchair.json

```diff
@@ -0,0 +1,13 @@
+{
+    "key": "highchair",
+    "type": "check",
+    "label": "Highchair",
+    "autoSuggestions" : false,
+    "customValues" : true,
```

Comment:
for `type=check` fields, these settings don't do anything, or am I overlooking something?